front/src/modules/ui/icon/index.ts

```diff
@@ -22,6 +23,7 @@ export {
   IconCalendarEvent,
   IconCheck,
   IconCheckbox,
+  IconChecks,
```

Comment:
Not sure this icon is used by the components you added? `IconCheck` is used instead.